mypy/main.py

```diff
@@ -1321,6 +1315,32 @@ def add_invertible_flag(
         dest="special-opts:files",
         help="Type-check given files or directories",
     )
+    return parser, strict_flag_names, strict_flag_assignments
+
+
+def process_options(
```

Comment:
Amazing, this func that contains some real logic is only 200 LOC now, much better than 1000+ to scroll!